main.go

```diff
@@ -32,6 +34,25 @@ func main() {
 		fmt.Printf("usage: %s path\n", os.Args[0])
 		os.Exit(1)
 	}
+
+	if *template_flag {
+		templ, err := ioutil.ReadFile(flag.Args()[0])
+		if err != nil {
```

Comment:
@vidriduch I am not sure if panicking with the error here is the best thing to do -- it does not really give the user much info on what went wrong and how to fix that.